SpatialGDK/Documentation/content/tutorials/multiserver-shooter/tutorial-multiserver-intro.md

```diff
@@ -19,21 +19,33 @@ Following this tutorial, you implement cross-server remote procedure calls (RPCs
 _Image: Example of cross-server shooting using multiserver zoning functionality._ </br></br>
 
 The tutorial demonstrates that the workflows and iteration speed you’re used to as an Unreal developer are almost entirely unaltered by the GDK: it’s just like regular Unreal.
-</br>
-</br>
-</br>
-**Notes:**
 
-* Before starting this tutorial you need to complete the [Example Project set up guide]({{urlRoot}}/content/get-started/example-project/exampleproject-intro). 
+### Change your branch of the Example Project 
+
+Before starting this tutorial you need to complete the [Example Project set up guide]({{urlRoot}}/content/get-started/example-project/exampleproject-intro). Following this, you clone or download and set up all the elements of the GDK and its dependencies, including the Example Project.
+
+To follow the Multiserver zoning shooter tutorial, you need to change your Example Project branch from `release` (which is the default) to `multiserver-tutorial-start`. To do this:
+
+1. If you have Unreal Editor open, close it.
+2. In GitHub, change your Example Project branch to `multiserver-tutorial-start`.</br>
+To do this, open a terminal window, navigate to directory containing your cloned Example Project  - this will be    `...\UnrealEngine\samples\UnrealGDKExampleProject` and enter `git checkout -b multiserver-tutorial-start`.
```

Comment:
```suggestion
To do this, open a terminal window, navigate to directory containing your cloned Example Project (this will be    `...\UnrealEngine\samples\UnrealGDKExampleProject`) and enter `git checkout -b multiserver-tutorial-start`.
```

I'm generally a fan of dashes, so was initially going to add a closing dash, but then I thought brackets would probably work better here, for readability.

Reply:
Also, just thinking, do we need to tell people how to change branch?

I'm thinking that:
- if they're using the GDK, they should know how to change branch already
- if we've included this explanation because we need them to do it this way specifically, we should make that clear, otherwise they'll just do it in the way they're used to (e.g. through the UI).

Reply:
It's a way to do this not the only way to do this.
I popped it in as hand-hold because it's a tutorial and Unreal is our evaluation platform, so we are likely to get noobs - like CEOs, journos and such-like,  having a go at this and we want to make it as easy as possible for them to see how it works.